l10n_br_stock_account_report/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# @ 2016 Kmee - www.kmee.com.br - Daniel Sadamo <daniel.sadamo@kmee.com.br>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Brazilian Localization WMS Accounting Report',
+    'category': 'Localisation',
+    'license': 'AGPL-3',
+    'author': 'Akretion, Odoo Community Association (OCA)',
```

Comment:
seems most of the code logic is from KMEE, so author should be KMEE instead of Akretion. LGTML otherwise.